cmd/samples/dsl/main.go

```diff
@@ -22,7 +22,13 @@ func startWorkers(h *common.SampleHelper) {
 		MetricsScope: h.Scope,
 		Logger:       h.Logger,
 	}
-	h.StartWorkers(h.Config.DomainName, ApplicationName, workerOptions)
+	worker := h.StartWorker(h.Config.DomainName, ApplicationName, workerOptions)
+	worker.RegisterWorkflow(SimpleDSLWorkflow)
```

Comment:
Workflow and activities should be registered before calling start.

Reply:
Yeah, I noticed this also. So none of these samples (besides `cron`) actually work now.